providers/base/api/handlers/mocks/api_data_handler.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery v2.30.1. DO NOT EDIT.
+// Code generated by mockery v0.0.0-dev. DO NOT EDIT.
```

Comment:
Shouldn't these version strings be correct? Are you using a source build of mockery locally or something?

Reply:
Yeah it was a source build.  I should probably just use the latest version we have in the `go.mod`